exporter/streams.go

```diff
@@ -111,6 +115,14 @@ func scanStreamGroupConsumers(c redis.Conn, stream string, group string) ([]stre
 	return result, nil
 }
 
+func ParseId(id string) float64 {
+	parsed_id, err := strconv.ParseFloat(strings.Split(id, "-")[0], 64)
```

Comment:
You might want to break out the `strings.Split(id, "-")[0]` and check that you actually have >= 1 resu;ts from the `split()`.

Reply:
I think you have to check for this, otherwise this has the ability to crash the exporter.

Reply:
I am looking at it, but cannot figure out the case in which it can crash the exporter. The item ID must be int and it cannot be 0. The func  strconv.ParseFloa returns 0 on strings. In which case it can crash the exporter?

Reply:
Oh you're right! Split() should always return at least one item here so you're fine, my bad.